src/libs/runtime-utils.js

```diff
@@ -70,8 +70,9 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
 }
 
 export const getGalaxyCost = app => {
-  return hourlyKubernetesAppCost(app) + persistentDiskCost({ size: 250 + 10 + 100, status: 'Running' })
-  // 250Gb for the NFS disk, 10Gb for the postgres disk, and 100Gb for the boot disk
+  // 2 * hourlyKubernetesAppCost because a Galaxy app has 2 nodepools by default
```

Comment:
How does this affect already-existing galaxy apps? Are those being updated to 2 nodepools as well, or will this show an inaccurate cost for them?

Regardless, I think we should ultimately try to surface this information in the app object, so we could compute it directly, rather than having to infer.